test/protobuf-net-data.Tests/protobuf-net-data.Tests.csproj

```diff
@@ -3,21 +3,48 @@
   <PropertyGroup>
     <DebugType Condition="$(codecov) == 'true'">full</DebugType>
     <IsPackable>false</IsPackable>
-    <TargetFramework>net471</TargetFramework>
+    <TargetFrameworks>net40;net47;net471;netcoreapp2.0</TargetFrameworks>
```

Comment:
Target framework net47 could be removed, right?

Reply:
it's possible ))